tests/test_utils/test_schemaview.py

```diff
@@ -544,8 +544,13 @@ def test_imports_relative():
         '../../L0_1/L1_1_0/index',
         '../../L0_1/cousin',
         '../L1_0_1/dupe',
-        './L2_0_0_0/child',
-        './L2_0_0_1/child',
+        'L2_0_0_0/child',
```

Comment:
merged impl ensures `./` prepended to relative child paths in the internal representation but doesn't require them in input

Reply:
Thanks. I am looking over the changes that were made because I'm suspicious that the "fix" (specifically, using `urlparse` and then checking for `scheme`) isn't working how people think it is, but rather as a side effect. `urlparse` is extremely lax in what it accepts and will parse; it does not validate whether or not what you give it is an URL.

Reply:
well, the only thing it's used for there is to check if there is a scheme component, so passing other strings is desired behavior? but yeah if you find some bug then by all means patch it